src/js/components/reviewData/ReviewDataComments.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * ReviewDataComments.jsx
+ * Created by Alisa Burdeyny 11/21/16
+ **/
+
+import React from 'react';
+import ReviewDataCommentsDropdown from './ReviewDataCommentsDropdown.jsx'
+import ReviewDataCommentsTextfield from './ReviewDataCommentsTextfield.jsx'
+
+import * as ReviewHelper from '../../helpers/reviewHelper.js';
+
+export default class ReviewDataComments extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            currentFile: "A",
+            fileComments: props.comments,
+            saveState: ""
+        };
+    }
+
+    changeFile(newFile) {
+        var fileComment = document.getElementById("submission-review-comments").value.trim();
```

Comment:
@cmc333333 is correct; we rarely directly touch the DOM in React because it is not guaranteed to exist (or exist in the way you think it will) when the code executes. One alternative would be to convert `ReviewDataCommentsTextfield` to be controlled by `ReviewDataComments` (ie, have the `textarea`'s value stored in `ReviewDataComments`'s state instead of in `ReviewDataCommentsTextfield`) so you can grab the text value out of the state instead of reading from DOM.

You _could_ use refs per CM's suggestion, but the fact that you're cloning props into state in child components suggests the current approach is an anti-pattern and that the master state should be in a higher component. Remember, you can flow changes upward to parent components via functions that are passed down through props, but prop data should represent data that isn't changed by the component itself.